### PR TITLE
(WIP) RSWW-10: simple CQRS accommodation service definiton

### PR DESCRIPTION
**Pull request by Michal299**
Creates the simple spring boot microservices repsonsible for serving the accommodation related functionalities. This is only very general service definition for now, will be used in the future as the base for the correct services.

cc. @Winetq @KingaW20

### CODE REVIEW

File: accommodation_service_commands/pom.xml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+		<java.version>17</java.version>
+		<h2.version>1.4.200</h2.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-amqp</artifactId>
+		</dependency>
+
```
Comment (Winetq) [inline accommodation_service_commands/pom.xml]: Maybe remove a new line here ;)

File: utils/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+version: "3.2"
```
Comment (Winetq) [inline utils/docker-compose.yml:1]: And bump this to 3.8?

**Review by Winetq — APPROVED** [review]
Cool 💪 And I have also two small suggestions.

**Comment by Michal299** [comment]
> Cool 💪 And I have also two small suggestions.

Thanks, but it is not finished yet XD
Im used to creating PRs just right after pushing the branch, even it is not “review ready”. This is the case as well, this is why I put (WIP) in the PR title and didn’t add anybody to the reviewers.

> WIP - work in progress 

But this is FYI only how I like working with git and remote repos. thanks for feedback, I will address your comments.

**Pull request merged**